src/libraries/System.Net.Http.Json/tests/FunctionalTests/System.Net.Http.Json.Functional.Tests.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="HttpClientJsonExtensionsTests.cs" />
+    <Compile Include="HttpContentJsonExtensionsTests.cs" />
+    <Compile Include="JsonContentTests.cs" />
+    <Compile Include="TestClasses.cs" />
+    <Compile Include="TranscodingReadStreamTests.cs" />
+    <Compile Include="TranscodingWriteStreamTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="$(CommonTestPath)System\Net\Capability.Security.cs">
+      <Link>Common\System\Net\Capability.Security.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonTestPath)System\Net\Configuration.cs">
+      <Link>Common\System\Net\Configuration.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonTestPath)System\Net\Configuration.Certificates.cs">
+      <Link>Common\System\Net\Configuration.Certificates.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonTestPath)System\Net\Configuration.Http.cs">
+      <Link>Common\System\Net\Configuration.Http.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonTestPath)System\Net\Configuration.Security.cs">
+      <Link>Common\System\Net\Configuration.Security.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonTestPath)System\Net\Http\LoopbackServer.cs">
+      <Link>Common\System\Net\Http\LoopbackServer.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonTestPath)System\Net\Http\LoopbackServer.AuthenticationHelpers.cs">
+      <Link>Common\System\Net\Http\LoopbackServer.AuthenticationHelpers.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonTestPath)System\Net\Http\GenericLoopbackServer.cs">
+      <Link>Common\System\Net\Http\GenericLoopbackServer.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonTestPath)System\Threading\Tasks\TaskTimeoutExtensions.cs">
+      <Link>Common\System\Threading\Tasks\TaskTimeoutExtensions.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetFrameworkCurrent)'">
+    <Compile Include="..\..\src\System\ArraySegmentExtensions.netstandard.cs" Link="ArraySegmentExtensions.netstandard.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\..\src\System\Net\Http\Json\TranscodingReadStream.cs" Link="TranscodingReadStream.cs" />
+    <Compile Include="..\..\src\System\Net\Http\Json\TranscodingWriteStream.cs" Link="TranscodingWriteStream.cs" />
```

Comment:
The networking libraries have generally been following a pattern where tests that build in product source are separated into a UnitTests directory/project separate from the FunctionalTests, e.g. https://github.com/dotnet/runtime/tree/master/src/libraries/System.Net.Http/tests/UnitTests